### DISCUSSION TITLE
Centering figures in TeX pdf (not working)

### DISCUSSION THREAD

**Discussion by 162348** [quarto-dev/quarto-cli#11637] [Q&A] [answered]
### Description

Consider the following minimal example:

````qmd

---
title: "Including svg images"
author: "Tester"
format:
  pdf:
    fig-format: svg
    fig-align: center
    keep-tex: true
---

```{r}
coplot(len ~ dose | supp, data = ToothGrowth, panel = panel.smooth,
       xlab = "ToothGrowth data: length vs dose, given type of supplement")
```

````

The output pdf contains one figure *without being centered*, although I declared `fig-align: center` in the header.

To make my point clearer, let us take a closer look. The figure is outputted by the corresponding TeX command, which is

```tex
\includegraphics{test_files/mediabag/test_files/figure-pdf/unnamed-chunk-1-1.pdf}
```

Adding `#| fig-align: center` at the beginning of the cell will wrap the command with the `center` environment:

```tex
\begin{center}
\includegraphics{test_files/mediabag/test_files/figure-pdf/unnamed-chunk-1-1.pdf}
\end{center}
```

However, this behaviour should already have been triggered by the global header YAML option.

Am I missing something here? Thanks in advance.

### Additional Information

Quarto: `1.6.39`
TinyTeX: `v2024.12`
MacOS: `15.1.1（24B91）`

**Comment by mcanouil** [comment, marked as answer]
- https://github.com/quarto-dev/quarto-cli/issues/6509

**Comment by 162348** [comment]
I'm sorry that I didn't find that issue already published.

Happy to find out this is behaviour somewhat by design:

> A "figure without a caption" is not a figure; it's an image.

**Comment by mcanouil** [comment]
> I'm sorry that I didn't find that issue already published.

All is fine. 😉
